### ISSUE TITLE
Missing the ability to add contacts from other third parties

### ISSUE THREAD

**Issue by lmag** [Dolibarr/dolibarr#29492]
### Feature Request

Missing 
![image](https://github.com/Dolibarr/dolibarr/assets/3288514/97815c07-6ccb-4215-bf8a-e4bcbb442297)
Like on project, contract, etc...

### Use case

Allow contact between company

### Suggested implementation

- [ ] Database
- [ ] Use dictionnaire
- [ ] Add interface like contract, project, proposal etc..

**Comment by JonBendtsen** [comment]
I think you can do this on proposals, where even if the Proposal belongs to one Thirdparty, then you can add a Contact from another Thirdparty to that Proposal

**Comment by FHenry** [comment]
@JonBendtsen that's already possible 

https://github.com/user-attachments/assets/99c24297-4da5-4d3e-a420-5e7c64573151

**Comment by hregis** [comment]
@FHenry I think what he means is to be able to add a contact from another company in the "contacts" tab of a third party, a bit in the spirit of the "contacts" tabs of quotes, invoices and others.

**Comment by FHenry** [comment]
@hregis Yes ! That what is missing, add contact from another thirdparty with a role can be a great feature

**Comment by JonBendtsen** [comment]
> @FHenry I think what he means is to be able to add a contact from another company in the "contacts" tab of a third party, a bit in the spirit of the "contacts" tabs of quotes, invoices and others.

yes, I also think that this is what @lmag  meant

**Comment by FHenry** [comment]
Yes that's it.
